src/main/java/io/temporal/internal/replay/ReplayClockContext.java

```diff
@@ -140,7 +141,7 @@ boolean isReplaying() {
     final OpenRequestInfo<?, Long> context = new OpenRequestInfo<>(firingTime);
     final StartTimerCommandAttributes timer =
         StartTimerCommandAttributes.newBuilder()
-            .setStartToFireTimeoutSeconds(delaySeconds)
+            .setStartToFireTimeout(Durations.fromSeconds(delaySeconds))
```

Comment:
the current method takes in delaySeconds as a long. If we wanted to, we can have it take a Java Duration instead, although perhaps not in this PR

Reply:
Let's make it Duration. As the API that calls it uses Duration and milliseconds.